homeassistant/components/template/light.py

```diff
@@ -292,6 +311,7 @@ def template_light_startup(event):
                 or self._color_template is not None
                 or self._white_value_template is not None
                 or self._availability_template is not None
+                or self._attribute_templates is not None
```

Comment:
I believe your default was `{}`, not `None`. So, this condition will never be false.